src/main/java/com/whalespottingjava/controllers/MembersController.java

```diff
@@ -1,17 +1,36 @@
 package com.whalespottingjava.controllers;
 
+import com.whalespottingjava.models.MemberDetails;
+import com.whalespottingjava.models.database.Member;
+import com.whalespottingjava.services.MemberDetailsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
 @Controller
 public class MembersController {
+    @Autowired
+    private MemberDetailsService memberDetailsService;
+
     @GetMapping("/members")
     public String getMembersPage() {
         return "members";
     }
 
     @GetMapping("/members/myaccount")
-    public String getMyAccountPage() {
+    public String getMyAccountPage(Model model) {
+        SecurityContext securityContext = SecurityContextHolder.getContext();
+        Authentication authentication = securityContext.getAuthentication();
+        MemberDetails memberDetails = (MemberDetails) authentication.getPrincipal();
+        Long memberId = memberDetails.getMember().getId();
+        Member member = this.memberDetailsService.loadMemberById(memberId);
+
+        model.addAttribute("member", member);
+
         return "my_account";
     }
 }
```

Comment:
Need new line at end of line

Reply:
There is one at the end of the file, git just doesn't show it. I believe the sign is the no-entry type symbol at the end of the file on the diff